src/test/authClass.login.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai'
+import * as sinon from 'sinon'
+
+import { AuthService } from '../apiServices/AuthService'
+import { JwtTokensInterface } from '..'
+
+describe('loginUser', () => {
+    const tokens: JwtTokensInterface = {
+        access: 'access',
+        refresh: 'refresh',
+    }
+
+    let loginUserStub: any
+    let AUTH
+    beforeEach(() => {
+        AUTH = new AuthService('user', 'password')
+        loginUserStub = sinon.stub(AUTH, 'login')
+    })
+
+    afterEach(() => {
+        loginUserStub.restore()
+    })
+
+    it('is a function', () => {
+        expect(loginUserStub).to.be.a('function')
+    })
+
+    it('should return access and refresh tokens', async () => {
+        loginUserStub.returns(tokens)
```

Comment:
I believe this should be `loginUserStub.resolves(tokens)` instead of `loginUserStub.returns(tokens)` for an async function.